### PR TITLE
Cran prep

### PR DESCRIPTION
**Pull request by jonathanbratt**

### CODE REVIEW

File: cran-comments.md [hunk 1]
```diff
@@ -1,14 +1,23 @@
-## v. 2.0.1
-* Resubmitting 2021-10-18
-* Updated to use the wordpiece.data and piecemaker packages.
+## v. 2.1.3
```
Comment (jonthegeek) [inline cran-comments.md]:
```suggestion
## v. 2.1.3 (Resubmission)
```

I read that explicitly saying it's a resubmission will speed up the process.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
One tiny change.

File: cran-comments.md (hunk 1 above)
Comment (jonathanbratt) [reply cran-comments.md]: ok!

**Review by jonthegeek — APPROVED** [review]
Looks good!

**Pull request merged**